apps/judicial-system/web/src/shared-components/Header/Header.spec.tsx

```diff
@@ -26,4 +37,31 @@ describe('Header', () => {
     // Assert
     expect(history.location.pathname).toEqual(Constants.REQUEST_LIST_ROUTE)
   })
+
+  test('should direct admin users to user overview route when user click the logo', async () => {
+    // Arrange
+    const history = createMemoryHistory()
+    history.push(Constants.USER_NEW_ROUTE)
+    const { getByTestId } = render(
+      <MockedProvider mocks={mockAdminQuery} addTypename={false}>
+        <Router history={history}>
+          <UserProvider>
+            <Header pathname={Constants.USER_LIST_ROUTE} />
+          </UserProvider>
+        </Router>
+      </MockedProvider>,
+    )
+
+    // wait for login to complete
+    await screen.findByTestId('logout-button')
+
+    const logo = getByTestId('link-to-home')
```

Comment:
This should be `screen.getByTestId`

Reply:
But this comes from the render function: const { getByTestId } = render(....